trestle/core/commands/merge.py

```diff
@@ -146,37 +138,22 @@ def merge(cls, effective_cwd: Path, element_path: ElementPath, trestle_root: Pat
                 f'Target model not found. Possibly merge of the elements not allowed at this point. {str(e)}'
             )
         target_model_path = effective_cwd / destination_model_alias
-        logger.debug(
-            f'look for target model path {target_model_path} at dest alias {destination_model_alias} rel to cwd'
-        )
+
         # target_model filename - depends whether destination model is decomposed or not
-        if target_model_path.exists():
-            logger.debug(
-                f'target model path does exist so target path is subdir with target alias {target_model_alias}'
-            )
-            target_model_path = target_model_path / target_model_alias
-        else:
-            logger.debug(f'target model filename does not exist so target path is target alias {target_model_alias}')
-            target_model_path = target_model_path / target_model_alias  # FIXME this is same as above
-        logger.debug(f'final target model path is {target_model_path}')
+        target_model_path = target_model_path / target_model_alias  # FIXME this is same as above
 
```

Comment:
Please see if that FIXME no longer applies or makes no sense.  If the code seems odd let me know and I want to check it before merge.  I can't see the context here.

Reply:
I revoked the changed for these lines, agree that all fixme should be checked. I will create an issue to go over all FIXMEs and see if they are relevant